Planner/src/Planner.java

```diff
@@ -114,15 +173,92 @@ public void actionPerformed(ActionEvent arg0) {
 		tasksList.addElement(task6);
 		tasksList.addElement(task7);
 		
+		Border border = BorderFactory.createLineBorder(Color.BLACK);
+		userLabel.setBorder(border); 
+		taskLabel.setBorder(border);
+		
+		listPanel.add(userLabel); 
 		listPanel.add(listWithUsers);
+		listPanel.add(Box.createRigidArea(new Dimension(0,20)));
+		listPanel.add(taskLabel);
 		listPanel.add(listWithTasks);
 		listPanel.setBackground(Color.white);
+		listPanel.setLayout(new BoxLayout(listPanel, BoxLayout.Y_AXIS));
+		listPanel.setAlignmentY(LEFT_ALIGNMENT); // correct alignment to the left (labels!) 
 		
 		// layout
 		topPanel.setLayout(new BorderLayout());
 		topPanel.setBackground(Color.white);
-		topPanel.add(panelButtons, BorderLayout.LINE_END);
+		topPanel.add(buttonPanel, BorderLayout.LINE_END);
+		topPanel.add(selectTaskButton, BorderLayout.LINE_START);
 		plannerFrame.add(topPanel, BorderLayout.PAGE_START);
 		plannerFrame.add(listPanel, BorderLayout.LINE_START);
+		
+		//show a dialog when we click on the user in User's list
+		listWithUsers.addListSelectionListener(new ListSelectionListener() {
+			public void valueChanged(ListSelectionEvent evtU) {
+				JTextField userName = new JTextField(25);
+			    JTextField userSurname = new JTextField(25);
+			    JPanel userPanel = new JPanel();
+			    userPanel.add(new JLabel("Edit name:"));
+			    userPanel.add(userName);
+			    userPanel.add(Box.createVerticalStrut(15)); // a spacer
+			    userPanel.add(new JLabel("Edit surname:"));
+			    userPanel.add(userSurname);
+			    userPanel.setLayout(new BoxLayout(userPanel, BoxLayout.Y_AXIS));
+				if (evtU.getValueIsAdjusting() == false) {
+					 int result = JOptionPane.showConfirmDialog(null, userPanel, 
+				               "Edit the User", JOptionPane.OK_CANCEL_OPTION);
+					 if (result == JOptionPane.OK_OPTION) {
+				         System.out.println("Name: " + userName.getText());
```

Comment:
Please add a space before all this lines that are inside the if.

Reply:
I have the spaces here. Everything inside the if is with a space.